src/renderer/mixins/formatter.js

```diff
@@ -34,7 +34,20 @@ export default {
      * @return {String}
      */
     formatter_date (value, format = null) {
-      dayjs.locale(window.navigator.userLanguage || window.navigator.language)
+      let userLanguage =
+        (window.navigator.userLanguage || window.navigator.language).toLowerCase()
+
+      if (userLanguage === 'en-us') {
+        userLanguage = 'en'
+      }
+
+      try {
+        require(`dayjs/locale/${userLanguage}`)
+      } catch {
+        userLanguage = 'en'
+      }
```

Comment:
More generic:

```suggestion
      try {
        require(`dayjs/locale/${userLanguage}`)
      } catch {
        try {
          userLanguage = userLanguage.split('-')[0]
          require(`dayjs/locale/${userLanguage}`)
        } catch {
          userLanguage = 'en'
        }
      }
```

Reply:
This will also handle locales like `de-DE`, `it-IT` ...

Reply:
Also more expensive due to how require works inside of electron.

Reply:
Gimme a minute, I'll boil it down to one `require`

Reply:
Update above